### PR TITLE
EWPP-2703: Reduce cyclomatic complexity of ContactMessageForm

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: src/Form/ContactMessageForm.php [hunk 1]
```diff
@@ -92,29 +113,54 @@ public function save(array $form, FormStateInterface $form_state): void {
...
 
+  /**
+   * Sets redirect URL.
+   *
+   * @param \Drupal\contact\ContactFormInterface $contact_form
+   *   Contact form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   Form state.
+   */
+  protected function setRedirectUrl(ContactFormInterface $contact_form, FormStateInterface $form_state) {
```
Comment (nagyad) [inline src/Form/ContactMessageForm.php:163]: : void

**Review by nagyad — CHANGES_REQUESTED** [review]

File: src/Form/ContactMessageForm.php (hunk 1 above)
Comment (yenyasinn) [reply src/Form/ContactMessageForm.php]: fixed

**Review by sergepavle — APPROVED** [review]

**Review by brummbar — CHANGES_REQUESTED** [review]
If we are trying to add better extensibility, the newly protected methods should have better parameters. Now they take the minimum set needed for our current scenario, but won't be of much value for other non-ewcms cases.
To check with @imanoleguskiza

**Comment by imanoleguskiza** [comment]
Let's rename the PR to better state what it does: Reduce cyclomatic complexity of ContactMessageForm

**Review by sergepavle — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**